android/sourcemaps.gradle

```
@@ -72,6 +57,23 @@ task uploadSourcemaps() {
     }
 }
 
+boolean isUploadSourcemapsEnabled() {
+    def envValue = System.getenv('INSTABUG_SOURCEMAPS_UPLOAD_DISABLE')?.toBoolean()
+    def defaultValue = true
+       
+    if(rootProject.hasProperty('instabugUploadEnable')) {
```

Just a tiny formatting suggestion.

```suggestion
    if (rootProject.hasProperty('instabugUploadEnable')) {
```